### PR TITLE
Adding Twitter access info

### PR DESCRIPTION
**Pull request by mauroservienti**
Please @adamralph / @sfarmar review

### CODE REVIEW

**Comment by adamralph** [comment]
:+1: LGTM

I'll just add some minor copy-edit comments...

File: README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
 * SQL Server
   * You must create the database before running the apps (see the connection string in the config files)
 
+### HashBus.Twitter.Monitor configuration [Optional]
+
+The `HashBus.Twitter.Monitor.Simulator` allows to test all the HashBus functionalities without using the public Twitter API/Feed. In order to run against the public Twitter API/Feed the following steps must be accomplished:
```
Comment (adamralph) [inline README.md]: > allows to test all the HashBus functionalities

prefer:

> allows testing of all HashBus functionality

Comment (adamralph) [inline README.md]: > public Twitter API/Feed

prefer:

> [Twitter API](https://dev.twitter.com/overview/documentation)

(Only hyper-linked on first occurrence.)

Comment (adamralph) [inline README.md]: > following steps must be accomplished

prefer:

> following steps are required

File: README.md [hunk 2]
```diff
@@ -4,6 +4,17 @@
...
+
+The `HashBus.Twitter.Monitor.Simulator` allows to test all the HashBus functionalities without using the public Twitter API/Feed. In order to run against the public Twitter API/Feed the following steps must be accomplished:
+
+* Go to https://apps.twitter.com/ and create a new application;
+* On the machine where the `HashBus.Twitter.Monitor` needs to run create the following `environment variables`:
+   * `HASHBUS_TWITTER_CONSUMER_KEY`: the created Twitter app consumer key;
+   * `HASHBUS_TWITTER_CONSUMER_SECRET`: the created Twitter app consumer secret;
+   * `HASHBUS_TWITTER_ACCESS_TOKEN`: the created Twitter app access token;
+   * `HASHBUS_TWITTER_ACCESS_TOKEN_SECRET`: the created Twitter app token secret;
+
```
Comment (adamralph) [inline README.md]: I'd prefer to remove the semicolons.

Comment (adamralph) [inline README.md]: > needs to run create

I would add a comma

> needs to run, create

Comment (adamralph) [inline README.md]: > `environment variables`

I would drop the inline code formatting.

Comment (adamralph) [inline README.md]: > the created

I would drop the word "created"

Comment (adamralph) [inline README.md]: > Twitter app token secret

should be

> Twitter app access token secret

**Comment by adamralph** [comment]
OK, I'm all done.

**Comment by mauroservienti** [comment]
Don't know why but your English @adamralph is much better than mine... :smiley:
All good, I'll merge.

**Pull request merged**